ray-operator/config/samples/ray-service.autoscaler.yaml

```diff
@@ -1,106 +0,0 @@
-# Make sure to increase resource requests and limits before using this example in production.
```

Comment:
This YAML was used for testing. We currently use `rayservice.autoscaling.yaml` instead. Delete this YAML.